src/DateComparison.php

```diff
@@ -4,17 +4,49 @@
 
 namespace CultuurNet\CalendarSummaryV3;
 
+use Carbon\CarbonImmutable;
+use DateInterval;
 use DateTimeImmutable;
 use DateTimeZone;
 
 final class DateComparison
 {
-    public static function onSameDay(DateTimeImmutable $date1, DateTimeImmutable  $date2): bool
+    public static function onSameDay(DateTimeImmutable $date1, DateTimeImmutable $date2): bool
```

Comment:
I think this class needs extensive unit tests now. It is very important this works as expected because all formatters rely on it.

Reply:
Although most methods are new, so I would say it's in the scope 😉. But a ticket is fine.

Reply:
Hmm yes I overlooked that a lot of the new code uses this `onSameDay()` method underneath. But we can do it in a follow-up ticket/PR since the new code won't actually be used in SAPI3/UDB3 until we make a new release of this library and update the version in those apps, so there's not a lot of risk.